README.md

```diff
@@ -0,0 +1,372 @@
+I: Loading cluster 'rh-moa-test-cluster1'                                                                                                                                                               
+I: Loading identity providers for cluster 'rh-moa-test-cluster1'                                                                                                                                        
+To use GitHub as an identity provider, you must first register the application:                                                                                                                 
+? List of GitHub organizations or teams that will have access to this cluster: openshift-online                                                                                                 
+* Open the following URL: https://github.com/organizations/openshift-online/settings/applications/new?oauth_application%5Bcallback_url%5D=https%3A%2F%2Foauth-openshift.apps.rh-moa-test-cluster1.j9n4.s
+1.devshift.org%2Foauth2callback%2Fgithub-1&oauth_application%5Bname%5D=rh-moa-test-cluster1&oauth_application%5Burl%5D=https%3A%2F%2Fconsole-openshift-console.apps.rh-moa-test-cluster1.j9n4.s1.devshift.org   
+* Click on 'Register application'                                                                                                                                                               
+? Copy the Client ID provided by GitHub: &lt;my-github-client-id&gt;                                                                                                                                   
+? Copy the Client Secret provided by GitHub: &lt;my-github-client-secret&gt;                                                                                                           
+I: Configuring IDP for cluster 'rh-moa-test-cluster1'                                                                                                                                                   
+I: Identity Provider 'github-1' has been created. You need to ensure that there is a list of cluster administrators defined. See `moactl user add --help` for more information. To login into th
+e console, open https://console-openshift-console.apps.rh-moa-test-cluster1.j9n4.s1.devshift.org and click on github-1
+```
+
+The IDP can take 1-2 minutes to be configured within your cluster.
+
+Run the following command to verify that your IDP has been configured correctly:
+
+```
+$ moactl list idps --cluster rh-moa-test-cluster1
+NAME        TYPE      AUTH URL
+github-1    GitHub    https://oauth-openshift.apps.rh-moa-test-cluster1.j9n4.s1.devshift.org/oauth2callback/github-1
+```
+
+### Create a dedicated-admin user
+
+Run the following command to promote your Github user to dedicated-admin:
+
+```
+$ moactl create user --cluster rh-moa-test-cluster1 --dedicated-admins=rh-moa-test-user
+```
+
+At this point you should be able to login to your cluster using your Github ID.
```

Comment:
Well, they could login with a github user prior to this command, they just wouldn't be dedicated-admin yet.  So I think this sentence should be moved to the previous section.

Reply:
That's a better flow.

I didn't realize this. When someone connects an IDP to a moa cluster, do all the users in the IDP automatically get a role in the cluster? Which role is it?

Reply:
Yes, I believe it is the basic-user clusterrolebinding, which is created automatically because the default mapping method is "claim".